k2/csrc/ragged.h

```diff
@@ -316,8 +316,29 @@ struct Ragged {
     if (!is.eof() || is.fail())
       K2_LOG(FATAL) << "Failed to construct Ragged array from string: " << src;
   }
-  // Construct from context and string.  This uses delegating constructors, (a
-  // c++11 feature), and an explicitly constructed Ragged<T>
+
+  explicit Ragged(const std::vector<std::vector<T>> &vecs) {
```

Comment:
I would like to create a function, e.g. `CreateRagged2`, instead of doing this in a constructor

Reply:
I don't thinks it's a problem, because the construction of two-dimensional arrays may be used more frequently. But I'm ok with changing the function to `CreateRagged2`.

Reply:
But do you think it's better to write a static function within the `Ragged` class or a global function out of the class? @qindazhu

Reply:
But do you think it's better to write a static function within the `Ragged` class or a global function out of the class? @qindazhu

Reply:
gloabl function in ragged_ops.h, as this is an util function like RaageedShape2. I think this function will even not be used in C++ code, that's another reason I suggest we don't put in class Ragged.

Reply:
Sure, it's more useful in python level.